code/modules/WVM/wvm.dm

```diff
@@ -589,6 +589,41 @@ GLOBAL_VAR_INIT(vendor_cash, 0)
 		new /datum/data/wasteland_equipment("Great Khan pelt boots",		/obj/item/clothing/shoes/f13/military/khan_pelt,					15)
 		)
 
+/obj/machinery/mineral/wasteland_vendor/denvendor
+	name = "Den Illegal Supply"
+	icon_state = "generic_idle"
+	prize_list = list(
+		new /datum/data/wasteland_equipment("Type93 Assault Rifle",			/obj/item/gun/ballistic/automatic/type93,							150),
+		new /datum/data/wasteland_equipment("Pulse Grenade",				/obj/item/grenade/empgrenade,										50),
+		new /datum/data/wasteland_equipment("Throwing Star",				/obj/item/throwing_star/ninja,								75),
+		new /datum/data/wasteland_equipment("Throwing Knife",				/obj/item/melee/onehanded/knife/throwing, 							50),
+		new /datum/data/wasteland_equipment("Flashbang",					/obj/item/grenade/flashbang, 										100),
+		new /datum/data/wasteland_equipment("Smoke Grenade",				/obj/item/grenade/smokebomb, 										100),
+		new /datum/data/wasteland_equipment("Tear Gas Grenade",				/obj/item/grenade/chem_grenade/teargas,								300),
+		new /datum/data/wasteland_equipment("MK. 23",						/obj/item/gun/ballistic/automatic/pistol/mk23, 						95),
+		new /datum/data/wasteland_equipment("Stinger Grenade",				/obj/item/grenade/f13/stinger,										35),
+		new /datum/data/wasteland_equipment("HE Grenade",					/obj/item/grenade/syndieminibomb/concussion, 						50),
+		new /datum/data/wasteland_equipment("Frag Grenade",					/obj/item/grenade/syndieminibomb/concussion/frag, 					35),
+		new /datum/data/wasteland_equipment("C4",							/obj/item/grenade/plastic/c4, 										85),
+
+		)
+	highpop_list = list(
+		new /datum/data/wasteland_equipment("Type93 Assault Rifle",			/obj/item/gun/ballistic/automatic/type93,							150),
+		new /datum/data/wasteland_equipment("Pulse Grenade",				/obj/item/grenade/empgrenade,										50),
+		new /datum/data/wasteland_equipment("Throwing Star",				/obj/item/throwing_star/ninja,								75),
+		new /datum/data/wasteland_equipment("Throwing Knife",				/obj/item/melee/onehanded/knife/throwing, 							50),
+		new /datum/data/wasteland_equipment("Flashbang",					/obj/item/grenade/flashbang, 										100),
+		new /datum/data/wasteland_equipment("Smoke Grenade",				/obj/item/grenade/smokebomb, 										100),
+		new /datum/data/wasteland_equipment("Tear Gas Grenade",				/obj/item/grenade/chem_grenade/teargas,								300),
+		new /datum/data/wasteland_equipment("MK. 23",						/obj/item/gun/ballistic/automatic/pistol/mk23, 						95),
+		new /datum/data/wasteland_equipment("Stinger Grenade",				/obj/item/grenade/f13/stinger,										35),
+		new /datum/data/wasteland_equipment("HE Grenade",					/obj/item/grenade/syndieminibomb/concussion, 						50),
+		new /datum/data/wasteland_equipment("Frag Grenade",					/obj/item/grenade/syndieminibomb/concussion/frag, 					35),
+		new /datum/data/wasteland_equipment("C4",							/obj/item/grenade/plastic/c4, 										85),
```

Comment:
why are all these so hilariously cheap to the point where any shitter can spawn in and buy one near instantly

Reply:
> why are all these so hilariously cheap to the point where any shitter can spawn in and buy one near instantly

These were set to this for testing, the outlaw gangtool explosives are also being looked at because they're a bit low, nerfing the low costs of the gangtool is absurd

also note this has been a draft for the last half a day or so, so don't expect it to be as intended.